packages/@dcl/dcl-rollup/api-extractor.ts

```diff
@@ -35,6 +35,17 @@ export async function apiExtractor(
   prepareOptions.configObject.mainEntryPointFilePath = path.resolve(
     prepareOptions.configObject.mainEntryPointFilePath
   )
+  const entryPoint = prepareOptions.configObject.mainEntryPointFilePath
+  if (!entryPoint.includes('/types/')) {
```

Comment:
Please add a comment about this corner case

Reply:
sure